readme.md

```diff
@@ -103,10 +113,25 @@ Write one Express route for each of four HTTP methods.
 Then, make each route respond with a one-word string containing the RESTful action that would most likely be associated with this route.
 
 ```js
+
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+app.post("/", function(res, req){
+  res.send("Post request to the homepage")
+});
+
+app.get("/", function(res, req){
+  res.send("Get request to the homepage")
+});
+
+app.put("/", function(res, req){
+  res.send("Update")
+});
+
+app.delete("/", function(res, req){
+  res.send("Delete")
+});
```

Comment:
This is generally correct, but we were looking for the responses to contain an appropriate restful action name:

* GET -> Show or Index (technically New and Edit use a GET verb as well, since they are just returning a form)
* POST -> Create
* PUT -> Update
* Delete -> Destroy